services/wallet/controllers_v4_test.go

```diff
@@ -627,3 +627,11 @@ func noOpMw() func(next http.Handler) http.Handler {
 		})
 	}
 }
+
+func signRequest(req *http.Request, publicKey httpsignature.Ed25519PubKey, privateKey ed25519.PrivateKey) error {
```

Comment:
After moving the wallet tests to wallet package we need to make this available in the controllers_v4_test.go test code.